example_test.go

```diff
@@ -32,3 +32,34 @@ func ExamplePublisher() {
 	// 2
 	// 1
 }
+
+func ExampleSubscriber() {
+	// ubsubscribing is done by canceling context, optional:
+	ctx, unsubscribe := context.WithCancel(context.Background())
+	defer unsubscribe()
+
+	sub := bps.NewInMemSubscriber(
+		map[string][]bps.SubMessage{
```

Comment:
any reason why you don't use `map[string]*bps.InMemTopic` instead?

Reply:
or even more abstract: `interface { Topic(string) *InMemTopic }`?

Reply:
to keep it short/simple, as it's a test helper anyway